src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts

```diff
@@ -392,6 +396,9 @@ export class TerminalTabbedView extends Disposable {
 				event.stopPropagation();
 			}
 		}));
+		this._register(dom.addDisposableListener(this._tabTreeContainer, dom.EventType.FOCUS_IN, () => {
+			this._terminalTabsFocusContextKey.set(true);
+		}));
```

Comment:
tried with just `FOCUS` and it wouldn't work. tested it though and `FOCUS_IN` seems like what we want. it's also used in other places.

Reply:
Need to reset the key on focus out as well